### ISSUE TITLE
Comment Quirks

### ISSUE THREAD

**Issue by Binary-Ape** [couchbase/query-ui#1]
Hello,

I've noticed while using the Query Workbench in 5.1 that there are a few issues with how comments are handled.

From the documentation ([N1QL Comments](https://developer.couchbase.com/documentation/server/current/n1ql/n1ql-language-reference/index.html)) it states that the supported comment type is a block comment (`/* This is a comment */`); however, if we are to use a double hyphen `--` then it will show the line as being commented out (turns all text green) but it doesn't respect that it is a comment and the query will fail.  

Another issue is that if you have a comment after the semi-colon it will complain that it can only run one query at a time:

`SELECT * FROM MyBucket LIMIT 1;
/* This is not allowed */`

Throws this error:

`{"error":"Error, you cannot issue more than onequery at once. Please remove all text after the semicolon closing the first query."}`

It would be useful to allow us to comment out multiple lines if we wish to do things like drop and re-create an index for testing purposes. We can just comment one line out instead of having to switch between two windows/page back and forth through history.

**Comment by ebenhaber** [comment]
Good point! I was able to add a fix for correctly handling block comments pretty easily, so that should work better in version 5.5. The issue with "--" non-comments was due to my copying this style from SQL, and no one has ever noticed that issue before! I'll remove that, but it's in a more restricted portion of the code, and won't make it out until a later release.

(And we do intend to better support multiple queries in a future release, but we're still working out the details of that.)

Thanks for your feedback!
